### DISCUSSION TITLE
page build failure

### DISCUSSION THREAD

**Discussion by divyanshushares** [community/community#45777] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The 404 error is showing up.
And with every commit a mail comes from github with following.

The page build failed for the `main` branch with the following error:

Unable to sync pages directory. Please try again later.

I am confused. Please help

**Comment by jge162** [comment]
Hello @divyanshushares 

You should check a couple settings. 

- Ensure that the repository name follows the format: `username.github.io` where username is your GitHub `username.`

- Make sure you are using the correct branch for your GitHub Pages site. 

- Go to the repository's settings, then to the GitHub Pages section, and ensure that the correct source branch and folder are selected.

Start there.
